WebARonARKit/WebARonARKit/Shaders.metal

```diff
@@ -27,152 +28,167 @@ typedef struct {
     float2 texCoord[[attribute(kVertexAttributeTexcoord)]];
 } ImageVertex;
 
-
 typedef struct {
     float4 position[[position]];
     float2 texCoord;
 } ImageColorInOut;
 
-
 // Captured image vertex function
-vertex ImageColorInOut capturedImageVertexTransform(ImageVertex in[[stage_in]])
-{
+vertex ImageColorInOut
+capturedImageVertexTransform(ImageVertex in[[stage_in]]) {
     ImageColorInOut out;
-
+    
     // Pass through the image vertex's position
     out.position = float4(in.position, 0.0, 1.0);
-
+    
     // Pass through the texture coordinate
     out.texCoord = in.texCoord;
-
+    
     return out;
 }
 
 // Captured image fragment function
-fragment float4 capturedImageFragmentShader(ImageColorInOut in[[stage_in]],
-                                            texture2d<float, access::sample> capturedImageTextureY[[texture(kTextureIndexY)]],
-                                            texture2d<float, access::sample> capturedImageTextureCbCr[[texture(kTextureIndexCbCr)]])
-{
-
-    constexpr sampler colorSampler(mip_filter::linear,
-                                   mag_filter::linear,
+fragment float4 capturedImageFragmentShader(
+                                            ImageColorInOut in[[stage_in]],
```

Comment:
I would recommend moving this whole block to the continuation indent (4 spaces) so it isn't all crammed over on the far right